integration/sawtooth_integration/docker/poet-smoke.yaml

```diff
@@ -0,0 +1,216 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+version: "2.1"
+
+services:
+
+  validator-0:
+    image: sawtooth-validator:$ISOLATION_ID
+    volumes:
+      - ../../../:/project/sawtooth-core
+    expose:
+      - 40000
+      - 8800
+    entrypoint: "bash -c \"\
+        ./bin/sawtooth admin keygen --force && \
+        ./bin/sawtooth config proposal create \
+          -k /etc/sawtooth/keys/validator.wif \
+          sawtooth.consensus.algorithm=poet \
+          sawtooth.poet.target_wait_time=5 \
+          -o config.batch && \
```

Comment:
Not certain, but you may also want to add call to `./bin/poet genesis ...` to create key pair for genesis validator.  Then you will also want to include that batch file in the `./bin/sawtooth admin genesis` below.